bucket/windterm.json

```diff
@@ -30,16 +30,16 @@
         "terminal"
     ],
     "checkver": {
-        "github": "https://github.com/kingToolbox/WindTerm",
-        "regex": "WindTerm_([\\d.]+)_Windows_Portable"
+        "url": "https://github.com/kingToolbox/WindTerm/releases",
+        "regex": "WindTerm_([\\d.]+)_Windows_Portable_x86_64.zip"
     },
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/kingToolbox/WindTerm/releases/download/$version/WindTerm_$version_Windows_Portable_x86_64.zip"
+                "url": "https://github.com/kingToolbox/WindTerm/releases/download/$majorVersion.$minorVersion.0/WindTerm_$version_Windows_Portable_x86_64.zip"
```

Comment:
Hardcoding a `0` like this might not be reliable. Instead, use something like this: https://github.com/ScoopInstaller/Main/blob/b66bb3f9634dd8f8fd8d835d62fe2dd8abcba2ab/bucket/clink.json#L19